Elemental/Components/AeTable/AeBaseTable.cs

```diff
@@ -8,119 +8,127 @@
+            }
+
+            return GridTemplateColumns;
         }
-        return "fad fa-sort";
-    }
 
-    // It's not recommended to use the RenderFragment builder yet so this is a catch-all workaround
-    protected virtual RenderFragment DefaultHeader(string header, int index){
-      throw new NotImplementedException("You must implement the DefaultHeader method for your Table Component.");
-    }
+        protected virtual void ToggleSortOnColumn(int column)
+        {
+            Console.WriteLine("ToggleSortOnColumn");
+            if (column == _sorting.column)
+            {
+                _sorting.sortAscending = !_sorting.sortAscending;
+            }
+            else
+            {
+                _sorting.column = column;
+                _sorting.sortAscending = true;
+            }
+            StateHasChanged();
+        }
 
-    protected virtual byte[] ExportToCSVInByte()
-    {
-        var builder = new StringBuilder();
-        (int column, bool sortAscending) _sorting = (0, true);
+        protected virtual string GetSortIconClass(int column)
+        {
+            if (column == _sorting.column)
+            {
+                return _sorting.sortAscending ? "fad fa-sort-down" : "fad fa-sort-up";
+            }
+            return "fad fa-sort";
+        }
 
-        foreach (var h in Headers)
+        // It's not recommended to use the RenderFragment builder yet so this is a catch-all workaround
+        protected virtual RenderFragment DefaultHeader(string header, int index)
         {
-            builder.Append(h + ",");
+            throw new NotImplementedException("You must implement the DefaultHeader method for your Table Component.");
         }
-        //Remove the last ,
-        builder.Length--;
-        builder.Append(Environment.NewLine);
 
-        foreach (var row in Dataset.OrderByDescending(data => Accessors[_sorting.column].Invoke(data)))
+        protected virtual byte[] ExportToCSVInByte()
         {
+            var builder = new StringBuilder();
+            (int column, bool sortAscending) _sorting = (0, true);
 
-            foreach (var (accessor, index) in Accessors.Select((a, i) => (a, i)))
+            foreach (var h in Headers)
             {
-                var value = accessor?.Invoke(row);
-                if (value.IndexOfAny(new char[] { '"', ',' }) != -1)
-                {
-                    builder.AppendFormat("\"{0}\"", value.Replace("\"", "\"\""));
-                    builder.Append(",");
-                }
-                else
-                {
-                    builder.Append(accessor?.Invoke(row) + ",");
-                }
+                builder.Append(h + ",");
             }
             //Remove the last ,
             builder.Length--;
             builder.Append(Environment.NewLine);
+
+            foreach (var row in Dataset.OrderByDescending(data => Accessors[_sorting.column].Invoke(data)))
```

Comment:
We should check to see if there's a sorting function instead of defaulting to the accessors here

Reply:
👋

Reply:
I changed this, then reverted it when I fixing a bug, then forgot to add it back... 😶